### ISSUE TITLE
TargetScan Prediction Score cutoff interpreation

### ISSUE THREAD

**Issue by PuMarianne** [KechrisLab/multiMiR#47]
Hi, 

I am trying to derive predicted targets for specific miRNAs. When looking at the targetScan results derived from the multiMir tools in comparision to the one derived from the targetScan website directly, I can not reproduce the results. Could you clarify how the cutoff for the prediction score is derived and how the ranking is done? 

Here is the code and results. multiMir v1.20.0
```
mir <- "mmu-miR-322-3p
predCutoff <- 70
get_multimir(mirna = mir, table   = "predicted", org = "mmu", predicted.cutoff = predCutoff)
```
database mature_mirna_acc mature_mirna_id target_symbol target_entrez     target_ensembl  score      type
1  targetscan     MIMAT0000549  mmu-miR-322-3p        Abi3bp        320712 ENSMUSG00000035258 -0.137 predicted
2  targetscan     MIMAT0000549  mmu-miR-322-3p          Rhoa         11848 ENSMUSG00000007815 -0.155 predicted
3  targetscan     MIMAT0000549  mmu-miR-322-3p          Ipo7        233726 ENSMUSG00000066232 -0.167 predicted
4  targetscan     MIMAT0000549  mmu-miR-322-3p        Pik3ca         18706 ENSMUSG00000027665 -0.168 predicted
5  targetscan     MIMAT0000549  mmu-miR-322-3p        Marcks         17118 ENSMUSG00000069662 -0.168 predicted
6  targetscan     MIMAT0000549  mmu-miR-322-3p        Elavl4         15572 ENSMUSG00000028546 -0.173 predicted
7  targetscan     MIMAT0000549  mmu-miR-322-3p         Ywhag         22628 ENSMUSG00000051391 -0.184 predicted
8  targetscan     MIMAT0000549  mmu-miR-322-3p         Fbxw7         50754 ENSMUSG00000028086  -0.21 predicted
9  targetscan     MIMAT0000549  mmu-miR-322-3p         Cpeb2        231207 ENSMUSG00000039782 -0.223 predicted
10 targetscan     MIMAT0000549  mmu-miR-322-3p           Sp3         20687 ENSMUSG00000027109 -0.224 predicted
11 targetscan     MIMAT0000549  mmu-miR-322-3p         Meis2         17536 ENSMUSG00000027210 -0.228 predicted
12 targetscan     MIMAT0000549  mmu-miR-322-3p        Tfap2b         21419 ENSMUSG00000025927 -0.237 predicted
13 targetscan     MIMAT0000549  mmu-miR-322-3p       Hnrnpdl         50926 ENSMUSG00000029328  -0.26 predicted
14 targetscan     MIMAT0000549  mmu-miR-322-3p          Gpc4         14735 ENSMUSG00000031119  -0.27 predicted
15 targetscan     MIMAT0000549  mmu-miR-322-3p         Kmt2e         69188 ENSMUSG00000029004 -0.274 predicted
16 targetscan     MIMAT0000549  mmu-miR-322-3p         Fgf13         14168 ENSMUSG00000031137 -0.277 predicted
17 targetscan     MIMAT0000549  mmu-miR-322-3p        Zfand4         67492 ENSMUSG00000042213 -0.288 predicted
18 targetscan     MIMAT0000549  mmu-miR-322-3p        Map2k4         26398 ENSMUSG00000033352 -0.291 predicted
19 targetscan     MIMAT0000549  mmu-miR-322-3p         Smap1         98366 ENSMUSG00000026155 -0.309 predicted
20 targetscan     MIMAT0000549  mmu-miR-322-3p        Ube2v2         70620 ENSMUSG00000022674  -0.31 predicted
21 targetscan     MIMAT0000549  mmu-miR-322-3p         Paip2         67869 ENSMUSG00000037058 -0.405 predicted
22 targetscan     MIMAT0000549  mmu-miR-322-3p          Nfyb         18045 ENSMUSG00000020248 -0.411 predicted

The scores are comparable for most of the targets but the ranks clearly differ (Cumulative weighted context++ score). 

TargetScan7 - https://www.targetscan.org/cgi-bin/targetscan/mmu_71/targetscan.cgi?mirg=mmu-miR-322-3p
filtered results only showing the targets predicted by multiMir

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/MARIAN~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/MARIAN~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link=blue vlink=purple>

rank | Target gene | Representative transcript | Representative miRNA | Cumulative weighted context++ score | Total context++ score
-- | -- | -- | -- | -- | --
179 | Nfyb | ENSMUST00000130911.2 | mmu-miR-322-3p | -0.38 | -0.41
214 | Ube2v2 | ENSMUST00000115777.4 | mmu-miR-322-3p | -0.35 | -0.52
237 | Map2k4 | ENSMUST00000046963.4 | mmu-miR-322-3p | -0.34 | -0.35
331 | Smap1 | ENSMUST00000027339.8 | mmu-miR-322-3p | -0.28 | -0.31
334 | Fgf13 | ENSMUST00000033473.6 | mmu-miR-322-3p | -0.28 | -0.3
373 | Cpeb2 | ENSMUST00000169035.2 | mmu-miR-322-3p | -0.27 | -0.27
382 | Kmt2e | ENSMUST00000094962.3 | mmu-miR-322-3p | -0.26 | -0.3
385 | Hnrnpdl | ENSMUST00000128187.2 | mmu-miR-322-3p | -0.26 | -0.28
388 | Sp3 | ENSMUST00000102689.4 | mmu-miR-322-3p | -0.26 | -0.27
447 | Zfand4 | ENSMUST00000036503.8 | mmu-miR-322-3p | -0.24 | -0.3
448 | Meis2 | ENSMUST00000110907.2 | mmu-miR-322-3p | -0.24 | -0.29
703 | Pik3ca | ENSMUST00000029201.8 | mmu-miR-322-3p | -0.19 | -0.2
757 | Elavl4 | ENSMUST00000106598.2 | mmu-miR-322-3p | -0.18 | -0.24
830 | Ipo7 | ENSMUST00000084731.3 | mmu-miR-322-3p | -0.17 | -0.19
876 | Marcks | ENSMUST00000092584.5 | mmu-miR-322-3p | -0.17 | -0.17
917 | Ywhag | ENSMUST00000055808.5 | mmu-miR-322-3p | -0.16 | -0.18
1061 | Gpc4 | ENSMUST00000033450.2 | mmu-miR-322-3p | -0.14 | -0.32
1086 | Rhoa | ENSMUST00000007959.8 | mmu-miR-322-3p | -0.14 | -0.15
1750 | Paip2 | ENSMUST00000041314.9 | mmu-miR-322-3p | -0.08 | -1.07
2454 | Tfap2b | ENSMUST00000027059.5 | mmu-miR-322-3p | -0.04 | -0.24
3879 | Fbxw7 | ENSMUST00000107679.2 | mmu-miR-322-3p | 0 | -0.21
3953 | Abi3bp | ENSMUST00000048471.8 | mmu-miR-322-3p | 0 | -0.14

</body>

</html>

**Comment by smahaffey** [comment]
@PuMarianne I agree I can't immediately see what's going on here.  Let me take a look and get back to you.
